vevestaX/vevesta.py

```diff
@@ -27,6 +27,7 @@ def __init__(self):
         self.__variables = {}
         self.__filename = self.get_filename()
         self.__sampleSize = 0
+        self.__skipEDA = False
```

Comment:
Name is self.speedUp = False

Reply:
the function have same name as variable so name should not be self.skillUp